### DISCUSSION TITLE
use esp32-cam with esp3d

### DISCUSSION THREAD

**Discussion by ddsmblal** [luc-github/ESP3D#1061] [General]
Hello everyone, thank you for this excellent work. I used this project on esp32-cam and everything went well, except that I did not succeed in adding a micro to operate the flash LED connected to pin 4.
 [ESP201] P<4> V<1>  dont work 
[ESP201] P4 V1 dont work 
?

**Comment by luc-github** [comment]
esp32cam is supported by esp3d 3.0 and correct syntax of [ESP201] is here https://esp3d.io/ESP3D/Version_3.X/documentation/commands/esp201/

You use wrong syntax - be noted that the pin used by the led is also used by the SDCard so doing this you will loose usage of SDCard of ESP32 Cam

**Comment by ddsmblal** [comment]
Thank you, the problem has been solved by adding a (=)
[ESP102] P=4 V=1

في الأحد، 24 نوفمبر 2024 4:29 ص Luc ***@***.***> كتب:
